Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PaalmanPingsMonteCarloAbsorption.py

```diff
@@ -118,16 +121,22 @@ def PyInit(self):
 
         # Shape Options
         self.declareProperty(name='Shape', defaultValue='FlatPlate',
```

Comment:
Should the default value be Preset?

Reply:
The default value did used to be Preset earlier on in this PR. In order to minimize the changes to the Indirect, Corrections UI I asked Tom yesterday to change this from Preset to FlatPlate because I thought using the preset shapes was more of an advanced option. @gvardany - what was your thinking? We could change it back again in the algorithm and the UI or just in the algorithm if you prefer Preset

Reply:
Flat plate needs more inputs, somehow preset feels as better default for the algorithm. This will also be more consistent with the numerical sibling. The situation is different from the GUI though; there is no way to preset the sample shape in the same GUI, one has to do it beforehand. So perhaps flat plate can be the default in the GUI?